pkg/estimator/pb/service.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto2";
+
+package github.com.karmada_io.karmada.pkg.estimator.pb;
+
+import "github.com/karmada-io/karmada/pkg/estimator/pb/generated.proto";
```

Comment:
Here, if we use path `github.com/karmada-io/karmada/pkg/estimator/pb/generated.proto`, we should keep karmada in `GOPATH`, otherwise the generate script cannot work. 
Should we change it to relative path `pkg/estimator/pb/generated.proto`? Although it is not correspond to `k/k` proto generated files, karmada source project also don't have to be in `GOPATH` when generating proto files.